.github/workflows/ci.yaml

```diff
@@ -55,6 +55,7 @@ jobs:
         env:
           HONEYCOMB_API_KEY: ${{ secrets.HONEYCOMB_API_KEY }}
           HONEYCOMB_DATASET: testacc
+          HONEYCOMB_COLUMN: duration_ms
```

Comment:
I'd rather we _not_ rely on the environment here: it looks like you're just mirroring the pattern put in place for the `honeycombio_datasets` datasource but datasets are a bit of a special case because we don't (generally) allow deletion of datasets to allow tests to create and cleanup after themselves.

I'd rather see the test for this datasource make a few columns at the start of the test and remove them at the end of the test. The test for the `honeycombio_recipients` datasource has a [decent example of this](https://github.com/honeycombio/terraform-provider-honeycombio/blob/main/honeycombio/data_source_recipients_test.go#L43-L56).

Reply:
Great points and example! fixing.